src/superphot_plus/model/classifier.py

```diff
@@ -275,22 +146,20 @@ def run(
         torch.backends.cudnn.deterministic = True
 
         (
-            train_data,
-            valid_data,
-            test_features,
-            test_classes,
-            test_names,
-            test_group_idxs,
-        ) = self.data
-
-        config_path = os.path.join(models_dir, f"superphot-config-{test_names[0]}.json")
+            train_dataset,
+            valid_dataset,
+        ) = self.train_data
+
+        config_path = os.path.join(models_dir, f"superphot-config-{run_id}.json")
         self.config.save(config_path)
-        model_path = os.path.join(models_dir, f"superphot-model-{test_names[0]}.pt")
 
-        train_iterator = DataLoader(train_data, shuffle=True, batch_size=BATCH_SIZE)
-        valid_iterator = DataLoader(valid_data, batch_size=BATCH_SIZE)
+        model_path = os.path.join(models_dir, f"superphot-model-{run_id}.pt")
```

Comment:
Is it worth saving this as a class variable, so you can get it later (to load models for testing)?

Reply:
I decided to save the best model state as a class variable instead. This way we won't write the whole model to disk and we can get right to testing without too much overhead.